portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Importer.java

```diff
@@ -26,10 +26,11 @@
     /**
      * Imports data.
      *
+     * @param jobId the current job id
      * @param authData authentication information
      * @param data the data
      * @return the operation result
      */
-    ImportResult importItem(A authData, T data); // REVIEW: The original throws IOException. Continue to use or return as part of the result?
+    ImportResult importItem(String jobId, A authData, T data); // REVIEW: The original throws IOException. Continue to use or return as part of the result?
```

Comment:
Can this be a UUID? I made it so we use UUID instead of String most places, see https://github.com/google/data-portability/pull/133

Reply:
I think so. That should be changed in a separate pull request,